.github/workflows/ci-docs.yml

```diff
@@ -0,0 +1,42 @@
+---
+# This workflow sets the test / all status check to success in case it's a docs only PR and ci.yml is not triggered
+# https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/defining-the-mergeability-of-pull-requests/troubleshooting-required-status-checks#handling-skipped-but-required-checks
+name: ci # The name must be the same as in ci.yml
+
+on:
+  pull_request:
+    paths-ignore: # This expression needs to match the paths ignored on ci.yml.
+      - '**'
+      - '!**/*.md'
+      - '!**/*.asciidoc'
+
+permissions:
+  contents: read
+
+jobs:
+  all:
+    runs-on: ubuntu-latest
+    steps:
+      - run: 'echo "No build required"'
+      # As long as this workflow uses the same name than .github/workflows/ci.yml and
+      # the test-reporter requires artifacts with the test results to exist.
+      # let's produce a dummy test so the test-report don't fail in this particular
+      # case
+      - name: Generate junit placeholder - to integrate with test-reporter
```

Comment:
I'm not 100% onboarded with this.

What about using `test-reporter` with empty values instead?

Reply:
Sure, I mimicked https://github.com/elastic/apm-agent-python/pull/1767. But I agree, setting it to allow empty sounds good.

Reply:
I was trying to find an option that allows empty test results, but I couldn't find it. Do you have a hint for me?